examples/dwarfdump.rs

```diff
@@ -161,16 +161,17 @@ fn add_relocations(
             object::RelocationKind::Absolute => {
                 match relocation.target() {
                     object::RelocationTarget::Symbol(symbol_idx) => {
-                        if let Some(symbol) = file.symbol_by_index(symbol_idx) {
-                            let addend = symbol.address().wrapping_add(relocation.addend() as u64);
-                            relocation.set_addend(addend as i64);
-                        } else {
-                            println!(
-                                "Relocation with invalid symbol for section {} at offset 0x{:08x}",
-                                section.name().unwrap(),
-                                offset
-                            );
+                        let symbol_result = file.symbol_by_index(symbol_idx);
+                        if let Err(_some_error) = symbol_result {
+                          println!(
+                              "Relocation with invalid symbol for section {} at offset 0x{:08x}",
+                              section.name().unwrap(),
+                              offset
+                          );
                         }
+                        let symbol = symbol_result.unwrap();
```

Comment:
This unwrap would panic when there was an error.

Reply:
updated